packages/plugin-awesome/tsconfig.json

```diff
@@ -3,6 +3,8 @@
   "include": ["./src"],
   "compilerOptions": {
     "jsx": "react",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "lib": ["ESNext"],
+    "types": ["node"]
```

Comment:
Root typescript config already includes the same `lib` and `types` fields. Please, rollback the changes because they're just duplicate existing fields